tubemq-server/src/test/java/org/apache/tubemq/server/broker/metadata/BrokerMetadataManageTest.java

```diff
@@ -1,74 +0,0 @@
-/**
```

Comment:
Why this file is deleted?

Reply:
I will check it right now , seems omit

Reply:
It's .gitignore that contains 'tubemq-server/src/test/',  omit BrokerMetadataManageTest.java